rust/data/presets.json

```diff
@@ -229,29 +225,31 @@
       "canTrickyDashJump",
       "canCWJ",
       "canHBJ",
-      "canUnmorphBombBoost",
+      "canDoubleBombJump",
       "canContinuousDboost",
       "can2HighWallMidAirMorph",
       "canPixelPerfectIceClip",
       "canBabyMetroidAvoid",
       "canBombJumpWaterEscape",
+      "canEscapeSand",
```

Comment:
This one is not too hard if you know how. I think that's more fitting for `VH`

Reply:
Actually, yeah.  I just know that most people can't do it, but that is more about not trying to learn, probably.